api/projects/views.py

```diff
@@ -106,7 +107,12 @@ def user_permissions(self, request: Request, pk: int = None):
         serializer = UserObjectPermissionsSerializer(instance=permission_data)
         return Response(serializer.data)
 
-    @action(detail=True, methods=["POST"], url_path="migrate-to-edge")
+    @swagger_auto_schema(responses={202: EmptySerializer()}, request_body=no_body)
```

Comment:
Oh nice, didn't know about the `no_body` util, that's cool! I wonder if there's something similar for the response and we can get rid of the EmptySerializer class entirely?